src/bento/adminData.js

```diff
@@ -21,10 +22,16 @@ export const VIEW = 'view';
 export const editViewPageTitle = 'Edit User';
 export const viewPageTitle = 'Approved Arm(s)';
 
+
 export const useMock = false;
 
+export const nodeName ="Node Name";  // Node Name configured in the Data Access Request form
+
+export const nodeField ="numberOfArms";  // Node field
+
+export const nodeLevelAccess = true;  //Node-Level Access is configured as "Off"
```

Comment:
the node field and nodeName can stay here but the NODE_LEVEL_ACCESS should be moved to the siteWideConfig.js file since it will be used in other locations.